hca/dss/__init__.py

```diff
@@ -361,15 +372,19 @@ def _download_manifest_bundle(self,
             logger.info('Primary copies of the files have been downloaded to `.hca` and linked '
                         'into per-bundle subdirectories of the current directory.')
 
-    def _download_tasks(self,
-                        bundle_uuid,
-                        replica,
-                        version="",
-                        download_dir="",
-                        metadata_files=('*',),
-                        data_files=('*',),
-                        num_retries=10,
-                        min_delay_seconds=0.25):
+    def _bundle_download_tasks(self,
+                               bundle_uuid,
+                               replica,
+                               version="",
+                               download_dir="",
+                               metadata_files=('*',),
+                               data_files=('*',),
+                               num_retries=10,
+                               min_delay_seconds=0.25):
+        """
+        Returns an iterator of tasks that each download one of the files in a bundle.
+        """
+        logger.info('Downloading bundle %s version %s ...', bundle_uuid, version)
         bundle = self.get_bundle(uuid=bundle_uuid, replica=replica, version=version if version else None)["bundle"]
```

Comment:
As we noticed today, we need to do `get_bundle.iterate(…)` here to get all files, not just the first 500. This API change is insidious in that way because it silently truncates the response to 500 files.